locale.c

```diff
@@ -7232,6 +7232,11 @@ S_emulate_langinfo(pTHX_ const int item,
 
         restore_toggled_locale_c(LC_TIME, orig_TIME_locale);
 
+        /* If strftime() returned an error, we return an empty string */
+        if (! temp) {
+            retval = "";
```

Comment:
This doesn't actually return. It keeps running the following blocks, which want to dereference `temp`.
Missing `break;` here?

Reply:
The `break` got lost between the box I tested with and the box I submitted with, and I'm emabarrsed I didn't notice